etl/steps/data/garden/unep/2023-12-12/renewable_energy_investments.meta.yml

```diff
@@ -1,56 +1,48 @@
+definitions:
+  common:
+    processing_level: minor
+    unit: 'billion US dollars'
+    short_unit: 'billion $'
```

Comment:
I wonder if it makes more sense to multiply the values by a billion either directly in the data or using the unit conversion factor, if that's available in the metadata?

This would allow you to shorten the short unit to '$', which I think would be desirable.